### PR TITLE
Servo tab: Rearrange mid, min, max and translate Rate:

### PR DESCRIPTION
**Pull request by HThuren**
Rearrange order to align with reciever tab, ie. min, middel, max
image

Translate possible of:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
I don't know what is the most expected form for servos, I don't use them, but seems reasonable to me.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
